app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -36,4 +64,132 @@ private EncryptorAdapter(final IServiceLocator serviceLocator) {
         return new ModuleWrapper<>(new EncryptorAdapter(serviceLocator));
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String hashWithoutSalt(final String pass) throws EncryptionException {
```

Comment:
The fact that it says password is because in practice this is about the only thing you can really use it for.